project/ProjectPlugin.scala

```diff
@@ -61,8 +61,9 @@ object ProjectPlugin extends AutoPlugin {
 
     lazy val coreSettings = Seq(
       libraryDependencies ++= Seq(
-        "com.47deg"      %% "github4s"       % "0.12.1",
+        "com.47deg"      %% "github4s"       % "0.13.0",
         "org.typelevel"  %% "cats"           % "0.9.0",
+        "joda-time"      % "joda-time"       % "2.9.9",
```

Comment:
Perhaps, now we've included joda-time, we can update [this](https://github.com/47deg/sbt-org-policies/blob/master/core/src/main/scala/sbtorgpolicies/utils.scala#L29) :P